### PR TITLE
Catching missing manual enrolment error

### PR DESCRIPTION
**Pull request by andrewhancox**
As of now it silently fails if the manual enrol plugin is missing, this adds a sensible error message.

### CODE REVIEW

**Comment by andrewhancox** [comment]
Has any thought been given to integrating this fix?

**Comment by andrewhancox** [comment]
Any thoughts about integrating this?

**Comment by bwalkerl** [comment]
Thanks @andrewhancox 

Apologies for the delay in getting around to these. I agree this should provide more info - just 2 things:

1. Error code 402 doesn't really seem appropriate for this.
2. It would be good if this also triggered a remote_view_error event  to add this to the logs, which was added in #67

**Comment by andrewhancox** [comment]
Done

File: error.php [hunk 1]
```diff
@@ -59,6 +59,14 @@
         ];
         $PAGE->set_title('402 Payment Required');
         break;
+    case 'nomanualenrolmentinstance':
+        $templatedata = [
+            'errorcode'    => 401,
+            'errortitle'   => get_string('errorpage_nomanualenrolmentinstancetitle', 'mod_scormremote'),
+            'errormessage' => get_string('errorpage_nomanualenrolmentinstancemessage', 'mod_scormremote'),
+        ];
+        $PAGE->set_title('402 Payment Required');
```
Comment (bwalkerl) [inline error.php]: This title is still set at 402.

Comment (andrewhancox) [reply error.php]: Sorted - careless of me...

**Pull request merged**